### PR TITLE
Add link to WebTransport WG in dependencies

### PR DESCRIPTION
**Pull request by dontcallmedom**

### CODE REVIEW

**Review by alvestrand — APPROVED** [review]

**Comment by aboba** [comment]
There isn't really a linkage between WebTransport and RTCDataChannel API (or WebSockets, for that matter).  The API paradigm is quite different as is the underlying protocol. So I'd delete that statement.

**Comment by lgrahl** [comment]
Arguably there was still inspiration and discussion based on the RTCDataChannel API (mostly on why it's not suitable). No strong opinion though.

File: webrtc-charter.html [hunk 1]
```diff
@@ -459,7 +459,7 @@ <h3 id="w3c-coordination">W3C Groups</h3>
 	<dl>
         <dt><a href="https://www.w3.org/media-wg/">Media Working Group</a></dt>
         <dd>The Media Working Group develops client-side media processing APIs and features that are likely to intersect with the WebRTC Working Group developments.</dd>
-        <dt class="todo">WebTransport Working Group</dt>
+        <dt><a href="https://www.w3.org/groups/wg/webtransport">WebTransport Working Group</dt>
         <dd>The WebTransport Working Group develops a client/server API that has been inspired by WebRTC datachannels and may inform their future evolution.</dd>
```
Comment (dontcallmedom) [inline webrtc-charter.html:463]:
```suggestion
        <dd>The WebTransport Working Group develops a client/server API that provides networking capabilities similar to how WebRTC datachannels have been used.</dd>
```
Does this work better @aboba ? if not, could you suggest an alternative wording? or are you suggesting we should not list the  WebTransport WG at all?

**Comment by dontcallmedom** [comment]
overtaken by events

**Pull request closed without merging**